docs/modules/ROOT/pages/guides/v4-migration/index.adoc

```diff
@@ -154,6 +154,7 @@ type Tech @node(label: "TechDB") @plural(value: "Techs") {
 
 There is no concept of a "main label" in the Neo4j database. As such, keeping these two separate arguments causes a disconnect between the database and the GraphQL library.
 As a result, the `label` and `additionalLabels` arguments have been condensed into a single argument `labels` which will accept a list of string labels that used when a node of the given GraphQL type is created.
+Please note that defining `labels` means you take control of the database labels of the node. Because indexes and constraints in the Neo4j database can only be assigned one label, for the moment the first label in the `labels` list will be used.
```

Comment:
Do we want to use the phrase "for the moment", here? I think if we write that then we should provide some context about why this is only temporary.

Reply:
I would remove "for the moment", this isn't going to change in the database AFAIK. A slightly nitpicky suggestion also not to start the second sentence with "because":

```suggestion
Please note that defining `labels` means you take control of the database labels of the node. Indexes and constraints in Neo4j only support a single label, for which the first element of the `labels` argument will be used.
```

Reply:
Hm the reason for the "for the moment" was the discussion about potentially adding an argument on @unique to reference the label for the constraint to be created on. Not sure how the general feeling is on this now, I believe it was due a discussion in the team.

Reply:
It would still be the case that we take the first element though - that wouldn't go away. This argument would just provide a way to override this behaviour.